configure.ac

```diff
@@ -443,6 +443,12 @@ AM_CONDITIONAL([HAVE_LIBFTD2XX], [test "x$have_libftd2xx" = xyes])
 AC_CHECK_HEADERS([curses.h ncurses/curses.h])
 AC_CHECK_LIB([ncurses], [initscr], [have_ncurses="yes"])
 AM_CONDITIONAL([HAVE_NCURSES], [test "x$have_ncurses" = xyes])
+# Defines libncurses_LIBS that we can use to link against ncurses
+# Needed since >=ncurses-6 where -ltinfo is needed in addition to -lncurses
+PKG_CHECK_MODULES(libncurses, [ncurses >= 5], [have_ncurses_pkgconfig="yes"],
```

Comment:
Can we initialise `have_ncurses_pkgconfig="no"` before PKG_CHECK_MODULES is run.

Reply:
I'd propose to replace the `true` in the next line with `have_ncurses_pkgconfig="no"` instead. okay for you?

Reply:
I was going to say we should be consistent, but I've seen a few other bits in configure where we do exactly that, so LGTM.